woocommerce/woocommerce-gutenberg-products-block
# Create Reviews by Product block placeholder

Part of #658.

Now that we are moving the _Reviews by Product_ block to be JavaScript-rendered, we need a placeholder while reviews load (loading reviews in the frontend will be implemented in a follow-up).

### Screenshots

![image](https://user-images.githubusercontent.com/3616980/60716161-f765f400-9f1e-11e9-9cbb-3268c4008e65.png)

### How to test the changes in this Pull Request:

1. In a new or existing post/page add a _Reviews by Product_ block.
2. Verify it creates a placeholder review.
